README.md

```diff
@@ -25,7 +25,7 @@
 
 ## Description
```

Comment:
We should have a separate section on "What's new in Ziggurat 3.0" or a section within "Description" should capture that. All the three keypoints should be mentioned here:

1) Introducing Middlewares - Here we should emphasize (briefly) on removal of dependency on protobuf for serialization. This is covered in Middleware section but a line or two would help anyone to catch a glimpse of the major changes.
2) Upgrading Kafka Streams from x to y
3) Changes in metrics format (we explain it a little and add that it is backward compatible)

Reply:
will add it as part of upgrade guide

Reply:
I'm planning to add individual documents for each feature and I don't think it makes sense to add it in the readme. The readme already has a lot of unnecessary information.

Reply:
@theanirudhvyas Ok. In that case there should be a PR for https://github.com/gojek/ziggurat/blob/master/UpgradeGuide.md too.